backend/question-service/server/app/utils/jsend.py

```diff
@@ -0,0 +1,18 @@
+
+from enum import Enum
+from fastapi.responses import JSONResponse
+
+
+class JSendStatus(Enum):
+    SUCCESS = "success"
+    FAIL = "fail"
```

Comment:
what's the difference between `FAIL` and `ERROR`? i thought that jsend only had 2 statuses success and fail - so the stuff I've implemented (API gateway, frontend validating API responses, user service), have only been returning/handling 'success' and 'fail'

my concern with 'introducing' `ERROR` means that the FE has to be modified to handle both fail and error

Reply:
Resolved in https://github.com/CS3219-AY2324S1/ay2324s1-course-assessment-g03/commit/6617a42ae09dab7ee6beaa7d478b25451f56c033:
- Changed to `FAIL` for errors with endpoints
- (Just for education purposes) JSend specifies the difference is in the message you are communicating to the client: something bad happened that shouldn't have happened ("error") or the server is working fine but your data isn't quite up to standards "fail" [(Source)](https://stackoverflow.com/questions/53199033/in-the-jsend-specification-what-is-the-difference-between-a-fail-and-an-error#:~:text=It's%20more%20about%20what%20you,standards%20(%22fail%22).)
- But for our application we can just stick to SUCCESS or FAIL